autossh/config.yaml

```diff
@@ -21,15 +21,20 @@ options:
   hostname: ''
   ssh_port: 22
   username: autossh
-  remote_forwarding:
-    - '127.0.0.1:8123:172.30.32.1:8123'
+  forward_local_ip_address: "172.17.0.1"
+  forward_local_port: 8123
+  forward_remote_ip_address: "127.0.0.1"
+  forward_remote_port: 8124
```

Comment:
Let's shift remote and local in their order? I'm not sure whether `forward_` is needed?